docs/wiki/concepts/Historical-Data.md

```diff
@@ -0,0 +1,146 @@
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Historical Buffers](#historical-buffers)
+- [Historical Range Access](#historical-range-access)
+
+## Historical Buffers
```

Comment:
Can we rename this page to Historical Buffers?
Historical data can be confused for historical data feeding into simulation runs